src/components/PopupFragment/PopupFragment.tsx

```diff
@@ -1,11 +1,12 @@
-import React, { useState, useEffect, useLayoutEffect } from 'react';
+import React, { ReactNode } from 'react';
 
 type Props = {
```

Comment:
wouldn't it be more recommended to specify of what these props are from? By naming it 'FragmentProps' or something alike instead of just 'Props'. I remember you giving me the same advice back when we started Dom Que Shot.

Reply:
Hmmmm ok then, great to know.